lib/api/controllers/remoteActions/dump.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-console */
+
+var
+  Promise = require('bluebird'),
+  path = require('path'),
+  fs = require('fs-extra'),
+  os = require('os'),
+  mkdirp = require('mkdirp'),
+  core = require('gcore').gcore,
+  RequestObject = require('kuzzle-common-objects').Models.requestObject,
+  moment = require('moment');
+
+var
+  _kuzzle;
+
+/**
+ * Create a dump
+ *
+ * @returns {Promise}
+ */
+function dump (request) {
+  var sufix = request && request.data.body.sufix ? '-' + request.data.body.sufix: '';
+  var dumpPath = path.join(
+    path.normalize(_kuzzle.config.dump.path),
+    moment().format(_kuzzle.config.dump.dateFormat).concat(sufix)
+  );
+
+  console.log('===========================================================================');
+  console.log('Generating dump in '.concat(dumpPath));
+  mkdirp.sync(dumpPath);
```

Comment:
We need to document the fact the user running Kuzzle needs some write permissions on this directory.